com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -185,6 +186,27 @@ internal set
         /// Gets if we are connected as a client
         /// </summary>
         public bool IsConnectedClient { get; internal set; }
+
+        private ClientObjMapNode<NetworkedClient, NetworkedObject> clientObjMapNode;
+
+        public ClientObjMapNode<NetworkedClient, NetworkedObject> ClientObjMapNode
+        {
+            get
+            {
+                if (clientObjMapNode == null)
+                {
+                    clientObjMapNode = new ClientObjMapNode<NetworkedClient, NetworkedObject>();
+                    clientObjMapNode.OnBypass = delegate(HashSet<NetworkedObject> results)
+                    {
+                        results.UnionWith(SpawnManager.SpawnedObjectsList);
+                    };
```

Comment:
this anonymous delegate could be replaced with one-line lambda function:
`clientObjMapNode.OnBypass = results => results.UnionWith(SpawnManager.SpawnedObjectsList);`

Reply:
👍